ui/raidboss/data/04-sb/raid/o1n.txt

```diff
@@ -0,0 +1,148 @@
+#~~~~~~~~~~~~~~~~~~~~~~#
+# DELTASCAPE V1.0--O1N #
```

Comment:
This is the tiniest of complaints, but maybe make this consistent with other timeline headers? I don't think anything uses ascii art boxes.